mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -490,7 +502,7 @@ const mapStateToProps = (state, ownProps) => {
   const { modelName } = ownProps;
   const { version } = ownProps.modelVersion;
   const tags = getModelVersionTags(modelName, version, state);
-  return { tags };
+  return { tags, modelSource: ownProps.modelVersion?.source };
```

Comment:
source is already in props (`ownProps`), you're mapping props to props here :) 
you can just use `this.props.modelVersion?.source` in line # 323